index.ts

```diff
@@ -83,6 +84,13 @@ const program = new Commander.Command(packageJson.name)
   A GitHub URL to use to bootstrap the shuttle backend with.
 `
     )
+    .option(
+    "--template <type>",
+    `
+  Use a premade Shuttle-provided template.
+    `
+ 
+)
```

Comment:
I am not sure what would be the difference between the `shuttle-example` and `template` flags.